core/src/main/java/cz/xtf/core/config/WaitingConfig.java

```diff
@@ -1,11 +1,22 @@
 package cz.xtf.core.config;
 
+import org.slf4j.event.Level;
+
 public class WaitingConfig {
 	public static final String WAITING_TIMEOUT = "xtf.waiting.timeout";
 
 	private static final String WAITING_TIMEOUT_DEFAULT = "180000";
 
+	public static final String WAITING_LOG_LEVEL = "xtf.waiting.log.level";
+
+	private static final Level WAITING_LOG_LEVEL_DEFAULT = Level.INFO;
+
 	public static long timeout() {
 		return Long.parseLong(XTFConfig.get(WAITING_TIMEOUT, WAITING_TIMEOUT_DEFAULT));
 	}
+
+	public static Level level() {
+		String level = XTFConfig.get(WAITING_LOG_LEVEL);
+		return (level == null) ? WAITING_LOG_LEVEL_DEFAULT : Level.valueOf(level.toUpperCase());
```

Comment:
There is method `XTFConfig.get(String property, String default)` that does the if part for you and hides the 'ugly' code. It'd be nicer to use that.

One liner would than look like:
`return Level.valueOf(XTFConfig.get(WAITING_LOG_LEVEL, WAITING_LOG_LEVEL_DEFAULT).toUpperCase() )`

Reply:
I saw that one, but I didn't want to type the WAITING_LOG_LEVEL_DEFAULT as String

Reply:
but it may work after all, let me try...

Reply:
I have to either 
```
return Level.valueOf(XTFConfig.get(WAITING_LOG_LEVEL, WAITING_LOG_LEVEL_DEFAULT.toString()).toUpperCase());
```
or change 
```
	private static final Level WAITING_LOG_LEVEL_DEFAULT = Level.INFO;
->
	private static final String WAITING_LOG_LEVEL_DEFAULT = "INFO";

```
since WAITING_LOG_LEVEL_DEFAULT I'm fine with making it String instead

Reply:
I'd go with `String` the same way timeout is `String`. They are properties in the end which are always strings by nature.

Reply:
Sure, no problem, it's in.